### PR TITLE
Fix the logging messages

### PR DESCRIPTION
**Pull request by petrkotas**
Fixes the #1057
Signed-off-by: Petr Kotas <petr.kotas@gmail.com>

---
This change is [Reviewable](https://reviewable.io/reviews/kubevirt/kubevirt/1074)

### CODE REVIEW

**Comment by petrkotas** [comment]
@rmohr I have fixed the logs.

File: pkg/virt-controller/watch/ovm.go [hunk 1]
```diff
@@ -373,7 +371,7 @@ func setupStableFirmwareUUID(ovm *virtv1.OfflineVirtualMachine, vm *virtv1.Virtu
 	}
 
 	vm.Spec.Domain.Firmware.UUID = types.UID(uuid.NewSHA1(firmwareUUIDns, []byte(vm.ObjectMeta.Name)).String())
-	logger.Infof("Setting stabile UUID '%s' (was '%s')", vm.Spec.Domain.Firmware.UUID, existingUUID)
+	logger.V(4).Infof("Setting stable UUID '%s' (was '%s')", vm.Spec.Domain.Firmware.UUID, existingUUID)
```
Comment (rmohr) [inline pkg/virt-controller/watch/ovm.go]: This log still does not make sense. Why would you want to print the "old" uuid, when we explicitly never override any given uuids?

File: pkg/virt-controller/watch/ovm.go [hunk 2]
```diff
@@ -466,7 +464,7 @@ func (c *OVMController) addVirtualMachine(obj interface{}) {
 			log.Log.Object(vm).Errorf("Cannot parse key of OVM: %s for VM: %s", ovm.Name, vm.Name)
 			return
 		}
-		log.Log.Object(vm).Infof("VirtualMachine created bacause %s was added.", vm.Name)
+		log.Log.Object(vm).Infof("VirtualMachine created because %s was added.", vm.Name)
```
Comment (rmohr) [inline pkg/virt-controller/watch/ovm.go]: Please give that a higher verbosity and also change the text: e.g. "Creation for VirtualMachine observed".

**Comment by fabiand** [comment]
Ping?

**Comment by petrkotas** [comment]
Yes, will fix it asap.

**Comment by petrkotas** [comment]
Done @rmohr sorry for the delay.

**Comment by petrkotas** [comment]
@rmohr @booxter did the rebase. Shall we proceed?

**Comment by petrkotas** [comment]
@rmohr sorry about that, I missed that while doing manual rebase (to many changes due to renaming).
Anyway, I have increased the verbosity to level 4.

**Comment by petrkotas** [comment]
@rmohr Can we merge this one?

**Pull request merged**